release/staging-build.yml

```diff
@@ -357,6 +376,15 @@ batch:
           PROJECT_PATH: projects/kubernetes/cloud-provider-vsphere
           CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/kubernetes.cloud-provider-vsphere
           RELEASE_BRANCH: 1-27
+    - identifier: kubernetes_cloud_provider_vsphere_1_28
```

Comment:
we dont need this component in the staging build. its not your fault its here, but would you mind adding the skip_staging_buildspec or whatever makefile var, check out one of the packages makefiles for the exact var. there are two to add.